src/persist/benches/consensus.rs

```diff
@@ -10,13 +10,16 @@
 //! Benchmarks for different persistent Write implementations.
 
 use std::path::Path;
+use std::sync::Arc;
 use std::time::{Duration, Instant};
 
 use criterion::measurement::WallTime;
 use criterion::{Bencher, BenchmarkGroup, BenchmarkId, Throughput};
+use tokio::runtime::Runtime as AsyncRuntime;
```

Comment:
i'm intentionally avoiding using the term Runtime in the new persist impl so that people outside the team are less confused by our naming, but it also means we don't have to rename all these anymore